server/apps/prepopulate/app_initialize.py

```diff
@@ -50,19 +54,20 @@ class AppInitializeWithDataCommand(superdesk.Command):
     """
 
     option_list = [
-        superdesk.Option('--entity-name', '-n', dest='entity_name', default='')
+        superdesk.Option('--entity-name', '-n', dest='entity_name', default=''),
+        superdesk.Option('--profile', '-p', dest='profile')
```

Comment:
Is not practical to be forced to execute a command for every file but to have it as an option can be useful.
Re: directory parameter, you should copy anyway the files to image, so why not to copy in the 'profile' directory?